app/Http/Controllers/QuizApiController.php

```diff
@@ -51,4 +51,28 @@ public function create(CreateQuizRequest $request): JsonResponse
             'message' => 'Quiz created',
         ], 201);
     }
+
+    public function update(CreateQuizRequest $request, Quiz $quiz): JsonResponse
+    {
+        $quiz->name = $request->name;
+        $quiz->description = $request->description;
+
+        $quiz->save();
+
+        if (!$quiz) {
+            return response()->json([
+                'message' => 'Quiz not found',
+            ], 404);
+        }
```

Comment:
Because you're using route model binding here, you don't actually need to check for the 404, Laravel will do it for you now.

This is great, however it does mean that the 404 laravel will automatically generate doesn't match what the front-end is expecting. There is a nice solution though, see my comment on the route :)